ui/shared/components/panel/search/LocusListItemsFilter.jsx

```diff
@@ -0,0 +1,81 @@
+import PropTypes from 'prop-types'
+import React, { useCallback } from 'react'
+import { Icon, Popup } from 'semantic-ui-react'
+import { BaseSemanticInput } from 'shared/components/form/Inputs'
+import { ColoredIcon } from 'shared/components/StyledComponents'
+import { PANEL_APP_CONFIDENCE_DESCRIPTION, PANEL_APP_CONFIDENCE_LEVEL_COLORS } from 'shared/utils/constants'
+import { camelcaseToTitlecase } from 'shared/utils/stringUtils'
+
+const PA_LABEL_HELP = 'A list of genes, can be separated by commas or whitespace.'
+const PA_ICON_PROPS = {
+  green: {
+    name: 'circle',
```

Comment:
since this is always the same it would be cleaner to make it a prop directly instead of a configuration